app/gcl/hands-on/terraform-gcp-challenge-lab-guide/page.css

```
@@ -0,0 +1,514 @@
+@import url('https://cdnjs.cloudflare.com/ajax/libs/tabler-icons/2.47.0/iconfont/tabler-icons.min.css');
```

Stylelint (17.14.1)

[error] Expected "url('https://cdnjs.cloudflare.com/ajax/libs/tabler-icons/2.47.0/iconfont/tabler-icons.min.css')" to be "'https://cdnjs.cloudflare.com/ajax/libs/tabler-icons/2.47.0/iconfont/tabler-icons.min.css'" (import-notation)

(import-notation)